docs/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork.md

```diff
@@ -2,8 +2,13 @@
 
 [Action Network](https://actionnetwork.org/) is an open platform that empowers individuals and groups to organize for progressive causes.
 
+The Action Network node allows you to automate work in Action Network and integrate Action Network with other applications. n8n has built-in support for a wide range of Action Network features, including creating new events, adding new people, tags, and signatures. The Action Network node allows you to create, get, update, and delete events, petitions, attendance, person, and tags in your Action Network account. 
+
 !!! note "Credentials"
-    You can find authentication information for this node [here](/integrations/builtin/credentials/actionnetwork/).
+    Refer to the [Action Network credentials](/integrations/builtin/credentials/actionnetwork/) for guidance on setting up authentication. 
+
+!!! note "Examples and templates"
+    For example, usage and templates to help you get started, take a look at n8n's [Action Network integrations](https://docs.n8n.io/integrations/builtin/credentials/actionnetwork/) list.
```

Comment:
This should link to the integrations list, not the credentials page. Please clarify with Polina where to find these lists.